service/src/main/java/bio/terra/policy/app/controller/TpsApiController.java

```diff
@@ -147,6 +147,13 @@ public ResponseEntity<ApiTpsPaoUpdateResult> linkPao(UUID objectId, ApiTpsPaoSou
     return new ResponseEntity<>(apiResult, HttpStatus.OK);
   }
 
+  @Override
+  public ResponseEntity<List<ApiTpsPaoGetResult>> listPaos(List<UUID> objectIds) {
+    return new ResponseEntity<>(
+        paoService.listPaos(objectIds).stream().map(ConversionUtils::paoToApi).toList(),
+        HttpStatus.OK);
```

Comment:
`ResponseEntity` has a nice helper constructor for `HttpStatus.OK`.
```suggestion
    return ResponseEntity.ok(paoService.listPaos(objectIds).stream().map(ConversionUtils::paoToApi).toList());
```